src/policy/feerate.cpp

```diff
@@ -8,6 +8,7 @@
 #include <tinyformat.h>
 
 const std::string CURRENCY_UNIT = "BTC";
+const std::string CURRENCY_ATOM = "SAT";
```

Comment:
Isn't "sat" usually lowercase?

Reply:
Yeah, hrm. The one place it's used right now is uppercase, though, so it may make sense to just do `ToLower(CURRENCY_ATOM)` in future uses. Or alternatively I use `ToUpper(CURRENCY_ATOM)` here...

Reply:
IMO `ToUpper(CURRENCY_ATOM)` makes more sense, since it could just as well be "cXBT" or such in some hypothetical sidechain or future hardfork.

Reply:
I am inclined to agree, but a lot of `ToUpper` was necessary: 4b304fefc517404bb109041cfcfa2a2c576c9a42

Reply:
IMO only the one case in `FeeModeFromString` is actually necessary.

Reply:
You mean if we mix-case the estimate modes? Yeah.